telegram/start_telegram_bot.js

```diff
@@ -191,16 +192,58 @@ module.exports = ({fs, id, limits, lnds, logger, payments, request}, cbk) => {
         });
       }],
 
+      //Get proxy agent
+      getProxyAgent: ['apiKey', 'saveKey', ({}, cbk) => {
+        //Exit early if not using a proxy
+        if(!proxy) {
+          return cbk();
+        }
+
+        const path = join(...[homedir(), home, proxyFile]);
```

Comment:
what is in this proxyFile

Reply:
how about --use-proxy <path> and the file at the path contains the details instead, I think if you want to securely store credentials that gives you more options

Reply:
sure, I can make that change.